packages/subgraph-service/contracts/libraries/IndexingAgreement.sol

```diff
@@ -577,7 +579,7 @@ library IndexingAgreement {
                     agreementId: params.agreementId,
                     collectionId: bytes32(uint256(uint160(wrapper.agreement.allocationId))),
                     tokens: expectedTokens,
-                    dataServiceCut: 0,
+                    dataServiceCut: params.indexingFeesCut,
```

Comment:
This will send the cut to the subgraph service contract, which would require a contract update for those funds to be used/transferred, right? (Not a problem per se, but maybe worth documenting)